docs/syntax_and_semantics/operators.md

```diff
@@ -219,6 +221,16 @@ proven faster than equality).
 | `>=` | greater or equal | `1 >= 2` | yes |
 | `<=>` | comparison | `1 <=> 2` | yes |
 
+#### Chaining Equality and Comparison
+
+The compiler allows for `==`, `!=`, `===`, `<`, `>`, `<=`, and `>=` 
+to be chained together. 
```

Comment:
This is a documentation of the language, not the compiler (that's just implementation), so it's better to not refer to compiler behaviour, but language behaviour. I'd also suggest adding more explanation to "chaining" together.

```suggestion
Equality and comparison operators `==`, `!=`, `===`, `<`, `>`, `<=`, and `>=` 
can be chained together and are interpreted as a compound expression. 
```